pages/partials/assessmentSyncErrorsAndWarnings.ejs

```diff
@@ -0,0 +1,21 @@
+<% if (assessment.sync_errors) { %>
+  <div class="alert alert-danger" role="alert">
+    <h2 class="h5 alert-heading">Sync error</h2>
+    <p>
+      There was an error syncing this assessment; the information you see below
+      may be inconsistent with this assessment's <code>infoAssessment.json</code> file.
+      Please correct the error and sync again.
```

Comment:
Would it be easy to add a link to the `edit` page for this `infoAssessment.json` file here? And similarly for course, courseInstance, etc.

Reply:
Yep, I think this would be pretty easy.